package.json

```diff
@@ -31,13 +31,13 @@
     "underscore": "^1.7.0",
     "body-parser": "^1.9.0",
     "validator": "^3.21.0",
-    "string": "^2.2.0"
+    "string": "^2.2.0",
+    "grunt-cli": "~0.1.13"
```

Comment:
that warning has been around for ages, doesn't actually affect the running of the grunt tasks, better to move it back into dev deps so that grunt-cli doesn't get installed in production instances